config/config.go

```diff
@@ -61,6 +61,12 @@ type Config struct {
 	// build files should be written to instead of RepoRoot.
 	WriteBuildFilesDir string
 
+	// GoProtoCompiler is the protocol buffers compiler to use for go code.
```

Comment:
These are Go-specific values. Please move to `goConfig` in `language/go/config.go`.